tilequeue/query/postgres.py

```diff
@@ -67,9 +64,15 @@ def __init__(self, sources, query_generator):
     def __call__(self, zoom, bounds):
         queries = []
         for source in self.sources:
-            if source.start_zoom <= zoom:
-                query = self.query_generator(source.template, bounds, zoom)
-                queries.append(query)
+            template_queries = []
+            for template_spec in source.template_specs:
+                if template_spec.zoom_start <= zoom < template_spec.zoom_stop:
```

Comment:
I've added #235 for deciding which makes more sense and then making all the code consistent. It seems like something that might need to be discussed more widely, and I don't want to hold this PR up.